tests/bluetooth/bsim_bt/bsim_test_eatt/tests_scripts/autoconnect.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
Maybe combine these test scripts that do basically the same thing? Make something more generic? Maybe outside the scope of this work?